sample-kotlin/build.gradle

```diff
@@ -35,7 +35,9 @@ dependencies {
     compile (project(':library')) {
         exclude module: "support-v13"
     }
-    compile project(':processor')
+
+    provided project(':processor')
 
     kapt project(':processor')
```

Comment:
@aurae ```kapt``` is enough for the compiler, but not enough for IDE. Android Studio marks with red unresolved references to annotations and ```PermissionRequest``` interface in code.